assembl/static2/js/app/components/administration/survey/save.js

```diff
@@ -0,0 +1,126 @@
+  const mediaImg = video.media ? video.media.img : null;
+  const mediaFile = getImageVariable(mediaImg, initialMediaFile);
+
+  const videoV = {
+    htmlCode: video.media ? video.media.htmlCode : '',
+    mediaFile: mediaFile,
+    titleEntries: video.title ? convertToEntries(video.title) : null,
+    descriptionEntriesBottom: video.descriptionBottom ? convertEntriesToHTML(convertToEntries(video.descriptionBottom)) : null,
+    descriptionEntriesSide: video.descriptionSide ? convertEntriesToHTML(convertToEntries(video.descriptionSide)) : null,
+    descriptionEntriesTop: video.descriptionTop ? convertEntriesToHTML(convertToEntries(video.descriptionTop)) : null
+  };
+  return videoV;
+}
+
+function getVariables(theme, initialTheme, order) {
+  const initialImg = initialTheme ? initialTheme.img : null;
+  const initialVideo = initialTheme ? initialTheme.video : null;
+  return {
+    identifier: 'survey',
+    titleEntries: convertToEntries(theme.title),
+    image: getImageVariable(theme.img, initialImg),
+    video: getVideoVariable(theme.video, initialVideo),
+    questions:
+      theme.questions &&
+      theme.questions.map(q => ({
+        id: q.id.startsWith('-') ? null : q.id,
+        titleEntries: convertToEntries(q.title)
+      })),
+    order: order
+  };
+}
+
+export const createMutationsPromises = (client: ApolloClient) => (
+  values: SurveyAdminValues,
+  initialValues: SurveyAdminValues
+) => {
+  const initialIds = initialValues.themes.map(t => t.id);
+  const currentIds = values.themes.map(t => t.id);
+  const idsToDelete = difference(initialIds, currentIds);
+  const idsToCreate = difference(currentIds, initialIds);
+
+  const allMutations = [];
+
+  const deleteMutations = idsToDelete.map(id =>
+    client.mutate({
+      mutation: deleteThematicMutation,
+      variables: { thematicId: id }
+    })
+  );
+  allMutations.push(...deleteMutations);
+
+  const createUpdateMutations = values.themes.map((theme, idx) => {
+    const initialTheme = initialValues.themes.find(t => t.id === theme.id);
+    const order = idx !== initialIds.indexOf(theme.id) ? idx + 1 : null;
+    const variables = getVariables(theme, initialTheme, order);
+    if (idsToCreate.indexOf(theme.id) > 0) {
+      return client.mutate({
+        mutation: createThematicMutation,
+        variables: variables
+      });
+    }
+    return client.mutate({
+      mutation: updateThematicMutation,
+      variables: {
+        id: theme.id,
+        ...variables
+      }
+    });
+  });
+  allMutations.push(...createUpdateMutations);
+  return allMutations;
+};
+
+export const save = async (mutationsPromises: MutationsPromises): Promise<SaveStatus> => {
+  let status = 'PENDING';
+  await runSerial(mutationsPromises)
+    .then(() => {
+      status = 'OK';
+      displayAlert('success', I18n.t('administration.successThemeCreation'));
+    })
+    .catch((error) => {
+      status = 'KO';
+      displayAlert('danger', error, false, 30000);
```

Comment:
In case of graphql error, error is an object. you should use `error.message` or use 
```
`${error}`
```